vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/verdict.rb

```diff
@@ -27,6 +32,16 @@ def has_result_code?(code)
         @result_code == code
       end
 
+      def translation_scope
+        "secure_rooms/access_rules.reasons"
+      end
+
+      private
+
+      def translate_reason(key)
+        return text(key) if key.present?
```

Comment:
No need for the `return`. The other option would be to add `default: ""` to the `text` call, but I'm fine with the `if`